include/deal.II/dofs/dof_tools.h

```diff
@@ -1760,6 +1760,25 @@ namespace DoFTools
                       const bool            invert_vertex_mapping  = false);
 
   /**
+   *  Same as above but allows boundary dofs on blocks to be excluded individually
+   *
+   * @arg <tt>exclude_boundary_dofs</tt>:  for each patch of cells around a
```

Comment:
Follow the style used elsewhere (which takes into account how doxygen renders this:
```
  @param variable_name Sentence that starts with a capital letter.
```